config/checkstyle/checkstyle-suppressions.xml

```diff
@@ -25,6 +25,7 @@
   <suppress files=".*[\\/]HelpfulSuggestions\.java" checks="MissingJavadocMethod"/>
 
   <!-- leniency for test files -->
+  <suppress files=".*[\\/]jib-testing-common[\\/].*\.java" checks ="MissingJavadocMethod"/>
```

Comment:
I wonder if we can start the pattern with `jib-testing-common[\\/]` without `."[\\/]`?

Reply:
I don't know if it matters, I guess this is a little more strict: `/jib-testing-common/` instead of `abc-jib-testing-common/`